app/models/room.rb

```diff
@@ -6,4 +6,7 @@ class Room < ApplicationRecord
 
   has_many :room_facilities, dependent: :destroy
   has_many :facilities, through: :room_facilities, source: :facility
+
+  # scope :latest, ->{order created_at: :desc}
+  scope :price_high_to_low, ->{order price: :desc}
```

Comment:
```suggestion
  scope :desc_price, ->{order price: :desc}
```
Như này cho ngắn gọn em ạ